docs/sources/panels-visualizations/visualizations/gauge/index.md

```diff
@@ -23,6 +23,8 @@ Gauges are single-value visualizations that can repeat a gauge for every series,
 
 {{< figure src="/static/img/docs/v66/gauge_panel_cover.png" max-width="1025px" alt="A gauge visualization">}}
 
+{{< docs/play title="Gauge Visualizations in Grafana" url="https://play.grafana.org/d/KIhkVD6Gk/" >}}
```

Comment:
I approved this already but I just noticed that the title text here doesn't match the dashboard title in Grafana Play. I had assumed that was your intent so I updated the time series PR and left comments on the bar and pie chart one. But now looking at this PR, I'm not sure. The suggestion below properly cases the generic phrase. If the point is to quote the UI, then this should be changed to: Grafana Gauge Visualization.
```suggestion
{{< docs/play title="gauge visualizations in Grafana" url="https://play.grafana.org/d/KIhkVD6Gk/" >}}
```